.github/workflows/cd.yml

```diff
@@ -11,8 +11,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2
-        with:
-          version: 8.2.0
```

Comment:
Are we sure if corepack is enabled in the CI/CD?

Reply:
See

https://github.com/pnpm/action-setup/blob/d2613e087f2e0aa841925861c5a5f7395d552177/README.md?plain=1#L11

Reply:
My concern was about enabling corepack, but it seems to be working anyway. https://github.com/dai-shi/waku/actions/runs/6620491644/job/17982999317